core/src/Root.ts

```diff
@@ -479,6 +479,8 @@ export class RootAPI implements Types.RootAPI {
                         rootAPI._forgetFocusedGrouppers,
                         rootAPI._autoRoot
                     );
+
+                    setTabsterOnElement(rootAPI._tabster, body, { root: rootAPI._autoRootInstance });
```

Comment:
Makes sense to add a cleanup for that in the dispose method above. To not leave the data attribute residue in the DOM when everything's disposed.

Reply:
I've added this to `Root` instance's `dispose` method instead of `RootAPI`. WDYT ?

Reply:
I suppose we need to go deeper™. For the consistency let's do setTabsterOnElement in the constructor (that also means removing it from  add too).

And perhaps let's remove the move() method completely. It's never used at this point. And if we're going to have a use case for it, we'll add a better solution depending on that use case.

Reply:
Agreed, it's looking a lot better now, only the `Root` instance uses `setTabsterOnElement` 👍👍👍